### PR TITLE
Suppress warnings for NPath complexity

### PR DESCRIPTION
**Pull request by evasileva-epam**
New variant for suppressing NPath complexity warning, as previous doesn't work.

### CODE REVIEW

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java  -10
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4500013&bid=15170031)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 import static org.apache.commons.lang3.StringUtils.isNotEmpty;
 
+@SuppressWarnings("all")
```
Comment (mf-code) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java]: it's bad to supress all warnings, why do we need to supress all?

Comment (mf-code) [reply jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java]: @SuppressWarnings("PMD.NPathComplexity")
@SuppressWarnings({"checkstyle:npathcomplexity", "checkstyle:cyclomaticcomplexity"})
possibly someone of that can help us

Comment (evasileva-epam) [reply jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java]: When I suppress all warnings complexity decreases only 10. The same was with attempt of suppressing for all class. I looked through code and don't see chance to suppress more functions used in getDriver() as it's read-only. So here is question for suppressing another way and deleting all my changes. It was discussed on stand up. 
To see result of suppress for each commit you could go https://app.codacy.com/.

**Comment by evasileva-epam** [comment]
@mf-code, I reverted changes as it was decided to supress NPath Complexity warning manually. 
You could merge this pull request or unmerge pull request for https://github.com/jdi-testing/jdi-light/pull/1322.

File: jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java (hunk 1 above)
Comment (RomE-HK) [reply jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java]: > 
> 
> @SuppressWarnings("PMD.NPathComplexity")
> @SuppressWarnings({"checkstyle:npathcomplexity", "checkstyle:cyclomaticcomplexity"})
> possibly someone of that can help us

Guess `@SuppressWarnings("PMD.NPathComplexity")` above the method should work right

**Comment by evasileva-epam** [comment]
According to  https://app.codacy.com/ variant "PMD.NPathComplexity" doesn't work at all. Remind that for the case "all" it was complexity smaller for 10.

**Comment by RomE-HK** [comment]
It worked but for some reason with time lag after Codacy checked the PR
image

**Comment by evasileva-epam** [comment]
@mf-code, do we want to see the result after merge or have I to revert changes?

**Pull request merged**